### PR TITLE
Zephyr: fix GATT/SR/GAS/BV-01-C

### PR DESCRIPTION
**Pull request by KKopyscinski**
We need to add two different services, so randomize UUID to avoid
duplication

### CODE REVIEW

**Comment by KKopyscinski** [comment]
This test also required patched `.ets` file from CASE0071492

**Comment by sjanc** [comment]
hmm could we make it predictable?  eg toggle registration of two known UUIDs?

**Comment by KKopyscinski** [comment]
Well, maybe add flag marking if wid 96 was run before? And register different UUID second time

**Comment by KKopyscinski** [comment]
The problem is that we do not register it in wid 96, we do it after it returns, that's why I made it this way. So we are limited to what we can do using `TestFunc()`

**Comment by KKopyscinski** [comment]
OK, we just need to register the same TestFunc twice, with different UUIDs, no need for randomization :)

**Review by sjanc — APPROVED** [review]

File: ptsprojects/zephyr/gatt.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 from ptsprojects.stack import get_stack
 from ptsprojects.zephyr.gatt_wid import gatt_wid_hdl, gatt_wid_hdl_no_write_rsp_check
 from ptsprojects.zephyr.gattc_wid import gattc_wid_hdl
+from random import randint
```
Comment (sjanc) [inline ptsprojects/zephyr/gatt.py:38]: so this is no longer needed, tight?

**Review by sjanc — CHANGES_REQUESTED** [review]

**Review by michal-narajowski — CHANGES_REQUESTED** [review]
Also needs a rebase and conflicts resolved.

**Comment by KKopyscinski** [comment]
@michal-narajowski Fixed and rebased

**Review by sjanc — APPROVED** [review]

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**